.NET/Microsoft.Recognizer.Text.TimeZone/English/UTCextractor.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Text.RegularExpressions;
+using Microsoft.Recognizers.Definitions.English;
+
+namespace Microsoft.Recognizers.Text.TimeZone.English
+{
+    class UTCextractor : BaseTimeZoneExtractor
+    {
+        internal sealed override ImmutableDictionary<Regex, string> Regexes { get; }
+
+        protected sealed override string ExtractType { get; } = Constants.SYS_TIMEZONE_UTC; // "Fraction";
+
+        private static readonly ConcurrentDictionary<string, UTCextractor> Instances = new ConcurrentDictionary<string, UTCextractor>();
+        public static UTCextractor GetInstance(string placeholder = "")
```

Comment:
Extractor and Parsers should only follow this strategy if tens of the same object are created during initialization. For other cases, please follow the regular model, like NumberRangeExtractor, for example.